{
  "Header": {
    "Original": "CVE-2022-22720 (Apache HTTP Server 2.4.52 and earlier fails to close inbound connectio ...)",
    "Line": 79447,
    "ID": "CVE-2022-22720",
    "Description": "(Apache HTTP Server 2.4.52 and earlier fails to close inbound connectio ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-2960-1}",
      "Line": 79448,
      "Type": "xref",
      "Bugs": [
        "DLA-2960-1"
      ]
    },
    {
      "Original": "- apache2 2.4.53-1",
      "Line": 79449,
      "Type": "package",
      "Package": "apache2",
      "Kind": "fixed",
      "Version": "2.4.53-1"
    },
    {
      "Original": "[bullseye] - apache2 2.4.53-1~deb11u1",
      "Line": 79450,
      "Type": "package",
      "Release": "bullseye",
      "Package": "apache2",
      "Kind": "fixed",
      "Version": "2.4.53-1~deb11u1"
    },
    {
      "Original": "[buster] - apache2 2.4.38-3+deb10u8",
      "Line": 79451,
      "Type": "package",
      "Release": "buster",
      "Package": "apache2",
      "Kind": "fixed",
      "Version": "2.4.38-3+deb10u8"
    },
    {
      "Original": "NOTE: https://httpd.apache.org/security/vulnerabilities_24.html#CVE-2022-22720",
      "Line": 79452,
      "Type": "NOTE",
      "Description": "https://httpd.apache.org/security/vulnerabilities_24.html#CVE-2022-22720"
    },
    {
      "Original": "NOTE: Fixed by: https://svn.apache.org/r1898692",
      "Line": 79453,
      "Type": "NOTE",
      "Description": "Fixed by: https://svn.apache.org/r1898692"
    }
  ]
}